content/en/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md

```diff
@@ -5,7 +5,18 @@ date: 2021-07-15
 slug: sig-usability-spotlight-2021
 ---
 
-**Author:** Kunal Kushwaha, Civo
+**Author:** Kunal Kushwaha (Civo)
+
+{{< note >}}
+SIG Usability, which is featured in this Spotlight blog, has been deprecated and is no longer active.
```

Comment:
```suggestion
SIG Usability, which is featured in this Spotlight blog, has been retired; it is no longer active.
```

Reply:
I believe we can go with 'deprecated and'

Reply:
OK. “Deprecated” is **definitely** wrong, but overall this PR is still clearly helping readers.